### ISSUE TITLE
KL increases

### ISSUE THREAD

**Issue by rojinsafavi** [pytorch/examples#631]
Hello,

I ran the VAE code, and I noticed that the KL term increases instead of decreasing. I was wondering if that is the expected behavior?

blue is the total loss, green is the binary cross-entropy loss, and red is the KL
![image](https://user-images.githubusercontent.com/19545198/65079381-dfb1df80-d953-11e9-999a-cd5bd82e3127.png)

**Comment by Coderx7** [comment]
Can you use the latest [PR ](https://github.com/pytorch/examples/pull/632)for VAE and see if this still happens ?

**Comment by PyExtreme** [comment]
@rojinsafavi , Is the problem still there in latest PR of VAE? I hope my inputs may help in this.

**Comment by anlijuncn** [comment]
Current master branch is Okay. KL is firstly increasing and then decreasing, if you train more epochs, you could find this trend. 
@PyExtreme @Coderx7 You could close it
